src/libraries/Native/Unix/System.Security.Cryptography.Native/pal_ssl.c

```diff
@@ -41,6 +46,24 @@ void CryptoNative_EnsureLibSslInitialized()
 #elif OPENSSL_VERSION_NUMBER < OPENSSL_VERSION_1_1_0_RTM
     EnsureLibSsl10Initialized();
 #endif
+
+    // Get length of session master key and generate one.
+    SSL_CTX* ctx = SSL_CTX_new(TLS_method());
+    if (ctx != NULL)
+    {
+        s_sessionTicketsMasterKeyLength = SSL_CTX_ctrl(ctx, SSL_CTRL_SET_TLSEXT_TICKET_KEYS, 0, NULL);
```

Comment:
Are we relying on the caller to ensure only one thread invokes this at a time?  What happens if that doesn't hold?  What happens if this shared state gets out of sync?